### PR TITLE
Added permissions `scw info`

### PR DESCRIPTION
**Pull request by QuentinPerez**
#191

### CODE REVIEW

File: pkg/commands/info.go [hunk 1]
```diff
@@ -59,9 +59,10 @@ func RunInfo(ctx CommandContext, args InfoArgs) error {
 			if err != nil {
 				return err
 			} else {
-				fmt.Fprintf(ctx.Stdout, "  [%d] %s\n", id, fingerprint)
+				fmt.Fprintf(ctx.Stdout, "  [%d] %s", id, fingerprint)
 			}
 		}
+		fmt.Fprintf(ctx.Stdout, "\n")
```
Comment (moul) [inline pkg/commands/info.go]: why this change ?

File: pkg/commands/info.go [hunk 2]
```diff
@@ -74,7 +75,20 @@ func RunInfo(ctx CommandContext, args InfoArgs) error {
 	fmt.Fprintf(ctx.Stdout, "  Images:\t\t%d\n", dashboard.ImagesCount)
 	fmt.Fprintf(ctx.Stdout, "  Snapshots:\t\t%d\n", dashboard.SnapshotsCount)
 	fmt.Fprintf(ctx.Stdout, "  Servers:\t\t%d\n", dashboard.ServersCount)
-	fmt.Fprintf(ctx.Stdout, "  Ips:\t\t\t%d\n", dashboard.IPsCount)
+	fmt.Fprintf(ctx.Stdout, "  Ips:\t\t\t%d\n\n", dashboard.IPsCount)
```
Comment (moul) [inline pkg/commands/info.go]: can you add the `\n` on a dedicated print to keep the code organized by sections ?

**Comment by moul** [comment]
LGTM

File: pkg/commands/info.go (hunk 1 above)
Comment (QuentinPerez) [inline pkg/commands/info.go]: To add a space between the two sections

File: pkg/commands/info.go [hunk 3]
```diff
@@ -59,9 +59,10 @@ func RunInfo(ctx CommandContext, args InfoArgs) error {
 			if err != nil {
 				return err
 			} else {
-				fmt.Fprintf(ctx.Stdout, "  [%d] %s\n", id, fingerprint)
+				fmt.Fprintf(ctx.Stdout, "  [%d] %s", id, fingerprint)
```
Comment (moul) [inline pkg/commands/info.go]: Why did you removed the '\n' ?

File: pkg/commands/info.go (hunk 1 above)
Comment (moul) [inline pkg/commands/info.go]: Ok, but why this: https://github.com/scaleway/scaleway-cli/pull/195/files#r40537290

File: pkg/commands/info.go (hunk 3 above)
Comment (QuentinPerez) [inline pkg/commands/info.go]: Because fingerprint has a already a '\n'

Comment (moul) [inline pkg/commands/info.go]: ok LGTM
